browsers.json

```diff
@@ -1,65 +1,65 @@
 {
-  "bs_ie_14_windows_10": {
+  "bs_edge_16_windows_10": {
```

Comment:
These changes are going to impact unit tests as well correct?

Reply:
The new browsers would, but the unit tests are still running successfully.